README.md

```diff
@@ -1717,6 +1717,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [WrapPixel](https://www.wrappixel.com/) — Download High Quality Free and Premium Admin dashboard template created with Angular, React, VueJs, NextJs and NuxtJs!
   * [Utils.fun](https://utils.fun/en) — All offline daily and development tools based on the browser's computing power, including watermark generation, screen recording, encoding and decoding, encryption and decryption and code formatting, are completely free and do not upload any data to the cloud for processing.
   * [QR Code Robot](https://qr-code-robot.com) — Free QR Code Generator. Offers possibilty to generate a QR Code for a webpage, sms, email, wifi auth or free text. Plans to offer also a generator for file sharing (pdf, mp3, video) and also a scan tracking of QR code.
+  * [Free Bio Link](https://freebio.dev) — Free bio link page for developers. You can made your own bio link (freebio.dev/username) by sending a pull-request and make your github more green 🟩🟩🟩.
```

Comment:
Nice, but this does not really have anything to do with the purpose of this list sorry.

Thank you!